blocks/components/editable/index.js

```diff
@@ -70,6 +70,8 @@ export default class Editable extends wp.element.Component {
 	}
 
 	render() {
-		return <div ref={ this.bindNode } />;
+		return wp.element.createElement( this.props.tagName, {
```

Comment:
Should we define a default `tagName` if none is specified, or should we assume that this must be specified by the block implementation?